Example/Podfile

```diff
@@ -4,3 +4,7 @@ use_frameworks!
 target 'ReusableDemo' do
   pod 'Reusable', :path => '../'
 end
+
+post_install do |installer|
+  system "sed -i '' -E 's/EMBEDDED_CONTENT_CONTAINS_SWIFT[[:space:]]=[[:space:]]YES/EMBEDDED_CONTENT_CONTAINS_SWIFT = NO/g' Pods/Target\\ Support\\ Files/Pods-ReusableDemo/*.xcconfig"
```

Comment:
Did you try using the `installer` parameter and its Ruby methods to manipulate the Build Settings, instead of modifying the xcconfig file yourself using `sed`?

It should give something like this (typed in GitHub so there might be some typo maybe):

```ruby
post_install do |installer|
    target = installer.pods_project.pod_targets.find { |t| in t.name == 'Pods-ReusableDemo' }
    target.build_configurations.each do |config|
       config.build_settings['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'NO'
    end
end
```